src/plenoptic/tools/stats.py

```diff
@@ -32,6 +32,42 @@ def variance(
     -------
     out
         The variance tensor.
+
+    See Also
+    --------
+    :func:`~plenoptic.tools.stats.skewness`
+        Calculate sample skewness.
+    :func:`~plenoptic.tools.stats.kurtosis`
+        Calculate sample kurtosis.
+
+    Examples
+    --------
+    >>> import torch
+    >>> from plenoptic.tools.stats import variance
+    >>> x = torch.tensor([[1.0, 2.0, 3.0], [3.0, 4.0, 5.0]])
+    >>> v = variance(x)
+    >>> v
+    tensor(1.6667)
+
+    If you have precomputed the mean, you can pass it and avoid recomputing it:
+
+    >>> precomputed_mean = torch.mean(x)
+    >>> v = variance(x, mean=precomputed_mean)
+    >>> v
+    tensor(1.6667)
+
+    If you want to compute along a specific dimension, you can specify it:
+
+    >>> v = variance(x, dim=0)
+    >>> v
+    tensor([1., 1., 1.])
+
+    This function differs from ``torch.var`` in that it does not apply a correction:
```

Comment:
```suggestion
    This function differs from the defaults of :func:`torch.var` in that it does not apply a correction:
```
get that intersphinx link